ffcx/codegeneration/definitions.py

```diff
@@ -189,12 +189,8 @@ def jacobian(self, e, mt, tabledata, num_points, access):
         return self._define_coordinate_dofs_lincomb(e, mt, tabledata, num_points, access)
 
     def cell_orientation(self, e, mt, tabledata, num_points, access):
```

Comment:
Is this even needed? I mean this is now too trivial, is it possible to eliminate any reference to this variable at first?

Reply:
I've removed cell_orientation